packages/forma-36-react-components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -125,6 +125,13 @@ export interface DropdownProps {
    * Defaults to `true`
    */
   usePortal?: boolean;
+  /**
+   * By passing down references from upstream components as part of this array,
+   * you will be able to disable automatic closing of the dropdown if you click
+   * somewhere else in the document besides the dropdown itself. See the
+   * Autocomplete component for an actual example of this usage.
+   */
+  nonClosingRefs?: RefObject<HTMLElement>[];
```

Comment:
Looking at this solution made me realize something
Our issue right now is that when we click outside the dropdow it will close it and the `toggleElement` is outside the dropdown.
So, I think we should pass a reference to the `toggleElement`. And only the `toggleElement`
Why? Because we make the API more restricted and simple... if in the future, for some reason, we need the dropdown to accept other elements that will not trigger closing it we can discuss this and extend

And now that I mentioned this alternative, I wonder if we really need a new prop or if we can just use the `toggleElement` prop to tell the hook what to do

Wdyt?

Reply:
Sounds great if we can make it work automatically, much easier for the developers 👍

Reply:
@denkristoffer small misunderstanding here. We still need to behaviour for element that trigger the dropdown programmatically from outside. But I think the point from @gui-santos is still a valid one to be considered for the toggle element itself.

Reply:
@anho But isn't the input used as `toggleElement`? 

> We still need to behaviour for element that trigger the dropdown programmatically from outside.

I don't quite follow you here. Could you explain when you would need this?